tox.ini

```diff
@@ -66,6 +66,7 @@ envlist =
     grpc_contrib-{py27,py34,py35,py36}-grpc
     httplib_contrib-{py27,py34,py35,py36}
     jinja2_contrib-{py27,py34,py35,py36}-jinja{27,28,29,210}
+    mako_contrib-{py27,py34,py35,py36}-mako{100}
```

Comment:
Does this patching also happen to work for `mako<1.0.0` by any chance?

Reply:
I will check the early version